Utils/gitlab_triggers/trigger_content_nightly_build.sh

```diff
@@ -60,5 +66,7 @@ source Utils/gitlab_triggers/trigger_build_url.sh
 curl "$BUILD_TRIGGER_URL" --form "ref=${_branch}" --form "token=${_ci_token}" \
     --form "variables[OVERRIDE_SDK_REF]=${DEMISTO_SDK_NIGHTLY}" \
     --form "variables[NIGHTLY]=true" \
+    --form "variables[IS_NIGHTLY]=true" \
```

Comment:
the addition of IS_NIGHTLY is because i'm trying to align the usage of this variable in the builds.